fbpic/particles/elementary_process/compton/numba_methods.py

```diff
@@ -174,91 +174,91 @@ def scatter_photons_electrons_numba(
                     cos_phi = 1.
                     sin_phi = 0.
 
-            # Loop through the number of scatterings for this electron
-            for i_scat in range(nscatter_per_elec[i_elec]):
```

Comment:
All the changes below consist in reindenting this `for` loop, so that it is located *within* the `if` condition `if nscatter_per_elec[i_elec] > 0:`. It was found that this fixes some compilation errors.